system/hardware/pc/hardware.py

```diff
@@ -53,6 +53,9 @@ def get_som_power_draw(self):
   def shutdown(self):
     print("SHUTDOWN!")
 
+  def set_display_power(self, on):
+    pass
```

Comment:
no need to implement this, just make it not abstract in the base class. think of it like an optional method